reframe/core/pipeline.py

```diff
@@ -673,6 +673,15 @@ def pipeline_hooks(cls):
     # FIXME: There is not way currently to express tuples of `float`s or
     # `None`s, so we just use the very generic `object`
 
+    #: Require that a reference is defined for each system that this test is
+    #: run on.
+    #:
+    #: :type: boolean
+    #: :default: :obj:`False`
```

Comment:
```suggestion
    #: :default: :class:`False`
```
I think this is typo, but I am not sure what the `class` keyword means in the other examples.

Reply:
Practically, they are rendered the same and `obj` is more correct as `False` is an object. In the past, I have started using `class` because that was it giving me the rendering that I wanted and didn't know that I could use `obj` too. However, now that I'm looking at the docs [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#role-py-obj), it should probably be `:const:` instead and perhaps we get also a link to the Python docs.

Reply:
Hm okay, I see. I tried the `:const:` and I got the same rendering and no link in the local docs. I guess const is a more accurate option but I am not sure if it will make a difference.

Reply:
No, it won't, but I can change it.